ql/src/java/org/apache/hadoop/hive/ql/exec/tez/HiveSplitGenerator.java

```diff
@@ -171,7 +171,7 @@ private void prepare(InputInitializerContext initializerContext) throws IOExcept
       // perform dynamic partition pruning
       if (pruner != null) {
         pruner.initialize(getContext(), work, jobConf);
-        pruner.prune();
+        pruner.prune(jobConf);
```

Comment:
note: `jobConf` was already shown to the `pruner` in the initialize

Reply:
Removed